ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -48,16 +49,21 @@ export default class DashboardQuickActionsCard extends Component {
           title: 'Secret path',
           subText: 'Path of the secret you want to read, including the mount. E.g., secret/data/foo.',
```

Comment:
This is't true, right? since the list of secrets here does not include the mount - it's just the `path`

Reply:
good catch! i just copied pasta's from the designs